include/m_npc_walk.h

```diff
@@ -2,10 +2,110 @@
 #define M_NPC_WALK_H
 
 #include "ultra64.h"
+#include "m_npc.h"
 
-typedef struct mNpc_walk_c {
+#define NPCW_GET_WALK_NUM(x) (int)((x) / 3)
+#define NPCW_MAX NPCW_GET_WALK_NUM(ANIMAL_NUM_MAX)
+
+typedef struct mNpcW_c {
     /* 0x00 */ char unk00[0x7C];
-} mNpc_walk_c; // size = 0x7C
+} mNpcW_c; // size = 0x7C
+
+typedef struct mNpcW_GoalData_c {
```

Comment:
Those struct names are made up or are known to be original?

Reply:
The convention is gotten from flashrom function names, but they're made up
I haven't checked if there's any struct names from walk in flashrom though, will check tomorrow morning

Reply:
It isn't in flashrom, so made up,  should I rename it to one that accommodates the guidelines?

Reply:
I think it would be better to rename them yeah